src/tools/JitOrderParser/JitOrderParser.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I think the general suggestion is to not check in sln files.

Reply:
OK